### PR TITLE
Initial version of cli

### PR DESCRIPTION
**Pull request by chemikadze**
commands
```
$ python -mqubell.cli 
Usage: __main__.py [OPTIONS] COMMAND [ARGS]...

Options:
  --tenant TEXT        Tenant url to use, QUBELL_TENANT by default
  --user TEXT          User to use, QUBELL_USER by default
  --password TEXT      Password to use, QUBELL_PASSWORD by default
  --organization TEXT  Organization to use, QUBELL_ORGANIZATION by default
  --help               Show this message and exit.

Commands:
  create_org
  export_app
  import_app
  list_apps
```
app export
```
$ python -mqubell.cli export_app --help
Usage: __main__.py export_app [OPTIONS] APPLICATION

Options:
  --recursive / --non-recursive  Export also dependencies.
  --output-dir TEXT              Output directory for manifest files, current
                                 by default.
  --help                         Show this message and exit.

```
app import
```
$ python -mqubell.cli import_app --help
Usage: __main__.py import_app [OPTIONS] [FILENAMES]...

Options:
  --help  Show this message and exit.
```
app listing
```
$ python -mqubell.cli list_apps --help
Usage: __main__.py list_apps [OPTIONS]

Options:
  --help  Show this message and exit
```
org creation
```
$ python -mqubell.cli create_org --help
Usage: __main__.py create_org [OPTIONS] ORGANIZATION

Options:
  --help  Show this message and exit
```

### CODE REVIEW

File: qubell/cli/__main__.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+from qubell.api.private.manifest import Manifest
+
+_platform = None
+
+
+@click.group()
+@click.option("--tenant", default="", help="Tenant url to use, QUBELL_TENANT by default")
+@click.option("--user", default="", help="User to use, QUBELL_USER by default")
+@click.option("--password", default="", help="Password to use, QUBELL_PASSWORD by default")
+@click.option("--organization", default="", help="Organization to use, QUBELL_ORGANIZATION by default")
```
Comment (dmakhno) [inline qubell/cli/__main__.py]: cannot parse help parameter, and default value.

**Comment by vasichkin** [comment]
Please add description to readme.

**Comment by chemikadze** [comment]
@vasichkin I've added usage to README and also made small reformatting.

File: qubell/cli/__main__.py (hunk 1 above)
Comment (chemikadze) [inline qubell/cli/__main__.py]: Real defaults come from `qubell.api.globals.QUBELL as qubell_config`, and these flags are just force-overrides for them.

**Comment by dmakhno** [comment]
46ea30f need to ensure that works fine with pypy and referenced in setup.py

**Comment by chemikadze** [comment]
@dmakhno Oh, thanks. There definitely are techniques to do that, but I don't think they worth time to spend, renamed to README back.

**Comment by vasichkin** [comment]
Cool.
Some thought (TODOs).
I'd like commands name was same as client methods. Abbreviated version could still exist, but full version is better.
Save env would be nice to have
restore_env.py could be merged here also.
(I can do that)

**Comment by chemikadze** [comment]
@vasichkin do you mean like
```
  create-org => create-organization
  export-app => ??? export-application ???
  import-app =>  ??? import-application ???
  init-ca => service
  list-apps => applications
```
?

While first three is ok as just nouns are written in long form, two last methods do not look self-descriptive in form they appear in code.

> Save env would be nice to have
> restore_env.py could be merged here also.

I implemented methods that needed the most, and never used save/restore env in real life, so probably I'm not best candidate for that.

**Comment by vasichkin** [comment]
create-organization
export-application 
import-application
init-common-services, see environment.init_common_service()

Ok to merge at this stage. I'll add rest later. Good starting point.

**Pull request merged**